paddle/fluid/inference/CMakeLists.txt

```diff
@@ -14,10 +14,6 @@ cc_library(paddle_fluid_api
 
 get_property(fluid_modules GLOBAL PROPERTY FLUID_MODULES)
 
-if(WITH_CONTRIB)
-  set(fluid_modules "${fluid_modules}" paddle_inference_api)
-endif()
-
 # Create static library
```

Comment:
This fix `segmentation fault`.
![image](https://user-images.githubusercontent.com/6836917/42672520-12d116e2-8699-11e8-9bae-3854a208b893.png)
The reason is that we add both `paddle_fluid` and `paddle_inference_api` in our `demo_ci/CMakeLists.txt`, and if `paddle_fluid` contatin `paddle_inference_api` again, it will segmentation fault.